contrib/mockkms/get_encryption_keys.go

```diff
@@ -27,40 +27,40 @@
 package main
 
 import (
-    "encoding/json"
```

Comment:
Since this PR is formatting some Go files that were previously un-formatted, perhaps this change should be done together with enforcing in CI that all Go files are always formatted?

Otherwise it will soon happen again, and every time we re-format whitespace it becomes quite difficult for reviewers to see if there was any (accidental or intentional) non-whitespace change in the affected code, which is a quite big security liability.

So I am proposing to either remove all the whitespace changes from this PR and put them in another PR which enforces formatting, or add in this PR something to CI which will enforce the formatting ([golangci-lint](https://github.com/golangci/golangci-lint) can do it, but there are probably also other GitHub actions for it).

Reply:
Should I focus on only the formatting, or there are other checks as well such as deadcode, errcheck, varcheck, should i add them to the ci pipeline as well.

Reply:
I'd say start with the formatting, adding other linters in further PRs will be easy enough. Otherwise this one will become too big and covering too many changes.

Reply:
ok i will start working with the basic formatter

Reply:
If you prefer you could undo the whitespaces changes in this PR so that we finish review+merge, then address the whitespace + formatter changes in a separate one.

Reply:
ok understood i will undo the whitespace changes in this PR and then updated it as well as, if it is ok i will open an issue for the golint-ci and create a PR for that as well so we can work on that seperately

Reply:
Sounds great, thanks!

Reply:
i have removed the unintentional whitespaces and have created an issue for the golint-ci
[https://github.com/apple/foundationdb/issues/11737](issue) just assign me the issue and i would start working on it

Reply:
@h4ck3r-04 all the whitespace formatting changes in this file `get_encryption_keys.go` are still there, can you please undo them as well?

Reply:
Ok, let me rephrase: this PR should not contain changes to formatting or whitespace, we will address that with #11737. Can you please undo the whitespace/formatting changes which are still in this PR?

Otherwise between this PR and the introduction of CI checks, whitespace may be changed again in some other PR.

Reply:
I have reverted the changes for all files, and made changes to the get_encryption_keys.go file

Reply:
Great, thanks.

Can you please fix the last 2 issues I found? See comments.

Reply:
fixed them, just check once

Reply:
Thanks; there are 2 remaining issues:
* https://github.com/apple/foundationdb/pull/11716#discussion_r1820587182
* https://github.com/apple/foundationdb/pull/11716/files#r1820584221 (pre-existing bug)